### PR TITLE
Tech Stack - Updated selenium.md with full simple tutorial

### PR DESCRIPTION
**Pull request by LanceUy**
Pull request for update on selenium.md #483. Updated with full tutorial example that goes through the basics of Selenium UI Automation as well as added a new folder containing image for examples.

### CODE REVIEW

File: Topics/Tech_Stacks/selenium.md [hunk 1]
```diff
@@ -124,6 +124,149 @@ driver.quit()
...
+    textField.click()
+    textField.send_keys("Hello")
+    ```
+3.  **Selenium: Selecting Multiple Choice**
+    
+    Let's say we wanted to select 'Option 2'. Following similar steps as above we will first locate the css selectors. Inspecting the multiple choice option for 'Option 2' we see the following:
+
+    ![alt text](SeleniumImages/multipleChoice.png)
+
+    We see there is a unique id field ```id = "i12"```which we can use for our css selector. We want to ensure that our css selectors are unique so that we choose only the item we want to select. As such below is the example code as how we could go about doing this.
```
Comment (harryye930) [inline Topics/Tech_Stacks/selenium.md:204]: I would suggest to add a bit more information on how to identify/choose what field to include in the CSS selector, as some of the fields can be dynamically generated as part of the JavaScript code.  I.e., The third option under a <div> with classname="something"

**Review by harryye930 — COMMENTED** [review]
I like your tutorial to automate a common web application, with such interactive guide and step-by-step process!

**Review by Maroosh-Gillani — COMMENTED** [review]
Great job! I am very impressed with your contribution.
I think adding the following changes can make your contribution even better:
- I can see that you have added multiple code fragments, but many of them are not explained explicitly. Adding an explanation will increase clarity.
- Consider adding a conclusion section that combines all the code sections into one.
- From my understanding, Selenium is very prone to errors, so this contribution could maybe benefit from adding a small debug section.

**Review by Maroosh-Gillani — APPROVED** [review]
Good job, looks good to me!

**Pull request merged**